ios/ImagePickerManager.m

```diff
@@ -164,13 +164,14 @@ -(NSMutableDictionary *)mapImageToAsset:(UIImage *)image data:(NSData *)data phA
         NSString *creationDate = [formatter stringFromDate:phAsset.creationDate];
         
         asset[@"timestamp"] = creationDate;
-        // Add more exif data here ...
+        asset[@"id"] = phAsset.localIdentifier;
+        // Add more extra data here ...
```

Comment:
Same comment as above. Does this need permissions to be accessed? I'm guessing as it is using PHAsset it does, right?

Reply:
yeah, needs permissions due to the PHAsset on iOS

Reply:
Makes sense, all is good here 🙌